optuna/samplers/_cmaes.py

```diff
@@ -94,6 +106,16 @@ def objective(trial):
             Note that the parameters of the first trial in a study are always sampled
             via an independent sampler, so no warning messages are emitted in this case.
 
+        restart_strategy:
+            Strategy for restarting CMA-ES optimization when converges to a local minimum.
+            If given :obj:`None`, CMA-ES will not restart (default).
+            If given 'ipop', CMA-ES will restart with increasing population size.
+            Please see also ``inc_popsize`` parameter.
```

Comment:
As commented on https://github.com/optuna/optuna/pull/1548#discussion_r463354929, how about using the experimental decorator to this newly introduced feature? For the docs, it is appropriate to note it as same as `consider_pruned_trials`:
```suggestion
        restart_strategy:
            Strategy for restarting CMA-ES optimization when converges to a local minimum.
            If given :obj:`None`, CMA-ES will not restart (default).
            If given 'ipop', CMA-ES will restart with increasing population size.
            Please see also ``inc_popsize`` parameter.
            
            .. note::
                Added in v2.1.0 as an experimental feature. The interface may change in newer
                versions without prior notice. See
                https://github.com/optuna/optuna/releases/tag/v2.1.0.
```